sdk/template/azure-sdk-template/pom.xml

```diff
@@ -34,9 +34,21 @@
       <version>1.6.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
nit: The Readme.md also seems to be outdated for the current Track 2

Reply:
I'll leave the Readme.md as-is for now. The readme has some guidance that can be useful for partners.